snap/local/bin/source-env-file.sh

```diff
@@ -17,7 +17,10 @@ logger "edgex service override: : SERVICE_ENV=$SERVICE_ENV"
 
 if [ -f "$SERVICE_ENV" ]; then
     logger "edgex service override: : sourcing $SERVICE_ENV"
-    source "$SERVICE_ENV"
+    set -o allexport
```

Comment:
Please update the logger to have the app name in prefix.
Also, rename SERVICE_ENV to something more meaningful.